lib/loggers/age.js

```diff
@@ -1,22 +1,28 @@
 const logger = require('../utils/logger');
+const errorLogger = require('../utils/error-logger');
 const { AgeEvent } = require('../db/models/age-event');
+const { isDbReady } = require('../utils/db');
 
 exports.startEvent = async ({ event, identifier }) => {
+	if (!isDbReady()) { return errorLogger.logDbNotReadyError(); }
```

Comment:
If the connection drops, `Mongoose` queries will never resolve, ie: `AgeEvent.find()` will never resolve, causing a timeout on the original request in the api

This fixes it, but returning early if the connection is dropped forever

Mongoose does have a timeout feature, but it requires an active connection (eg: https://stackoverflow.com/questions/29004445/how-do-i-set-a-timeout-on-a-mongoose-query).  Mongoose does not seem to have a timeout feature which kicks in when the connection is not active